### DISCUSSION TITLE
How to package partitioned datasets?

### DISCUSSION THREAD

**Discussion by davidgasquez** [datonic/hub#31] [Q&A]
Say we want to package a HIVE partitioned dataset. What is the best way to do that?

**Comment by TomNicholas** [comment]
Partitions are to tables as chunks are to arrays. So same solution as for https://github.com/orgs/datonic/discussions/29#discussioncomment-12123472

**Reply by davidgasquez** [reply]
Yes! That's a great one. I _think_ I was getting into how to allow folks to manage/define partitioned datasets and how that would affect other aspects. Should definitely merge and update these two issues.
